### DISCUSSION TITLE
How to handle exposed client keys in EMQX 5

### DISCUSSION THREAD

**Discussion by N0tronic** [emqx/emqx#10566] [Q&A]
Hi guys,

I search a possibility to handle leaked client private keys. In EMQX 4.4 this was possible:
https://www.emqx.io/docs/en/v4.4/security/crl-ocsp-stapling.html#crl

I don´t understand why this feature was removed in EMQX 5. Are there another way?

Cheers

**Comment by SergeTupchiy** [comment]
Hi @N0tronic,
This has been recently implemented in EMQX 5, please try the latest release [EMQX v5.0.24](https://github.com/emqx/emqx/releases/tag/v5.0.24). You can find a config example in `etc/emqx.conf.example`.
Recent EMQX Enterprise pre-releases also expose OCSP configuration to Dashboard UI: [EMQX Enterprise e5.0.3-alpha.3](https://github.com/emqx/emqx/releases/tag/e5.0.3-alpha.3).
